README.md

```diff
@@ -100,20 +104,20 @@ To pull down the latest version:
 
 There is presently an issue with `libxmlsec1` versions => `1.3.0` and Apple Silicon which results in an ungraceful failure in a required dependency `lxml` when `cpython` attempts to compile it. The following workaround has been tested successfully with a Macbook Pro M1 Pro and Python 3.10+. This involves downgrading `xmlsec` to a known working version (`1.2.37`).
 
-1. Install the required `xmlsec` dependencies above. 
+1. Install the required `xmlsec` dependencies above.
 2. Clone the repository locally with `git clone`.
 3. Create a virtual environment such as [virtualenv](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) or [Anaconda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html) using Python 3.9+ and activate it.
 4. `brew edit libxmlsec1` and replace the entire contents of the brew package with the following [gist](https://raw.githubusercontent.com/Homebrew/homebrew-core/7f35e6ede954326a10949891af2dba47bbe1fc17/Formula/libxmlsec1.rb)
 5. `brew unlink libxmlsec1` and `brew uninstall libxmlsec1`
 6. In your shell `rc` file (`.bashrc`, `.zshrc`, etc.), set the environment variable `HOMEBREW_NO_INSTALL_FROM_API` to `1`.
 7. Force a clean installation with `brew install /opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/libxmlsec1.rb`
 8. Trigger a clean installation of the pip package with `pip install . --no-cache-dir`.
-9. Remove `HOMEBREW_NO_INSTALL_FROM_API` from your shell `rc` file.
-10. (optional) To keep `brew` from upgrading the package, run `brew pin libxmlsec1`.
+9. Downgrade the version of `Werkzeug` from `3.0.1` to `2.2.2` using `pip install Werkzeug==2.2.2`.
```

Comment:
Can you copy this instruction to the Install from PyPI above and leave a note referencing the issue you opened?

Reply:
Done.